room_1:
  x: -4.742
  y: 1.409
  z: 0.694
  w: 0.720

room_2:
  x: -0.749
  y: 1.309
  z: 0.651
  w: 0.759

consultation_room:
  x: 2.184
  y: 1.124
  z: 0.662
  w: 0.749

room_3:
  x: -6.836
  y: -1.348
  z: -0.648
  w: 0.762

room_4:
  x: -2.797
  y: -1.368
  z: -0.757
  w: 0.653

room_5:
  x: 1.122
  y: -1.387
  z: -0.675
  w: 0.737

room_6:
  x: 5.192
  y: -1.298
  z: -0.721
  w: 0.693

lobby:
  x: 6.390
  y: 2.660
  z: 0.999
  w: -0.044

spawn:
  x: 0.000
  y: 0.000
  z: 0.000
  w: 1.000
